tests/integration/persist_docs/models/table_hudi_model.sql

```diff
@@ -0,0 +1,2 @@
+{{ config(materialized='table', file_format='hudi') }}
```

Comment:
Yes, you are right, since there were many iterations on this PR already, For now, I'll disable the model to keep it simple and merge this PR, later in the next iteration I'll bring back both these tests.